doc/source/whatsnew/v0.18.0.txt

```diff
@@ -192,6 +193,21 @@ Bug Fixes
 
 
 - Bug in ``.loc`` result with duplicated key may have ``Index`` with incorrect dtype (:issue:`11497`)
+
+
```

Comment:
need a doc-note, and you are changing things which are not relevant. e.g. pls rebase on master first.